File: Python/ex3.py
a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
less = []
more = []

for i in range (0, len(a)):
    if a[i] < 5:
        less.append(a[i])
    else:
        more.append(a[i])
print(less, more)

File: JS/ex3.js
var   a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
var less = []
var more = []

for (i = 0; i < a.length; i++)
    if (a[i] < 5) {
        less.push(a[i])
    }else {
        more.push(a[i])
    }console.log(less, more)